zebra-consensus/src/transaction.rs

```diff
@@ -293,7 +293,7 @@ where
             // (approximately 50,000 for a 2 MB transaction)
             let (utxo_sender, mut utxo_receiver) = mpsc::unbounded_channel();
 
```

Comment:
These changes will cause a merge conflict with PR #3415.
(And I'd like to keep or rephrase the comment about "basic checks" - it's important that only quick checks are done first, so slow checks are all done in parallel.)

Reply:
Sure, I'll revert these.